src/redux/checkin.test.js

```diff
@@ -301,35 +325,246 @@ describe('checkin: adding tasks', async (assert) => {
+    expected: createBaseBlockersTestState(),
+  });
+
+  assert({
+    given: 'adding multiple tasks to existing checkins by existing users',
+    should: 'add the tasks to the state',
+    actual: [
+      ...getInitialBlockerActions(),
+      addUser({ id: user2Id }),
+      addTeam({ id: team2Id, ownerId: user2Id }),
+      addBlocker({ description: 'Life gets in the way', checkinId: checkin.id, userId: owner.id }),
+      addCheckin({
+        id: checkin2Id,
+        userId: user2Id,
+        teamId: team2Id,
+        createdAt: checkin2CreatedAtTimestamp,
+      }),
+      addBlocker({ description: 'My dog ate my code', checkinId: checkin2Id, userId: user2Id }),
+      addBlocker({ description: 'My computer died', checkinId: checkin2Id, userId: owner.id }),
+    ].reduce(reducer, reducer()),
+    expected: createBaseBlockersTestState({
+      users: {
+        [owner.id]: owner,
+        [user2Id]: createTestUser({ id: user2Id }),
+      },
+      teams: {
+        [team.id]: team,
+        [team2Id]: createTestTeam({ id: team2Id, ownerId: user2Id, users: [user2Id] }),
+      },
+      checkins: [
+        createTestCheckin({
+          ...checkin,
+          blockers: [
+            createTestBlocker({
+              description: 'Life gets in the way',
+              checkinId: checkin.id,
+              userId: owner.id,
+            }),
+          ],
+        }),
+        createTestCheckin({
+          id: checkin2Id,
+          userId: user2Id,
+          teamId: team2Id,
+          createdAt: checkin2CreatedAtTimestamp,
+          blockers: [
+            createTestBlocker({
+              description: 'My dog ate my code',
+              checkinId: checkin2Id,
+              userId: user2Id,
+            }),
+            createTestBlocker({
+              description: 'My computer died',
+              checkinId: checkin2Id,
+              userId: owner.id,
+            }),
+          ],
         }),
-        createTestTask({ description: 'Push code to GitHub', checkinId: '2', userId: '2' }),
-        createTestTask({ description: 'Review GitHub PRs', checkinId: '2', userId: owner.id }),
       ],
     }),
   });
 });
+
+describe('checkin selectors: getUsersList', async (assert) => {
+  assert({
+    given: 'no users in state',
+    should: 'return an empty array',
+    actual: getUsersList(createTestState()),
+    expected: [],
+  });
+
+  assert({
+    given: 'users in state',
+    should: 'return an array or users',
+    actual: getUsersList(
+      createTestState({
+        users: {
+          '1': createTestUser({ id: '1', name: 'Bob' }),
+        },
+      })
+    ),
+    expected: [{ id: '1', name: 'Bob' }],
+  });
+});
+
+describe('checkin selectors: getTeamsList', async (assert) => {
+  assert({
+    given: 'no teams in state',
+    should: 'return an empty array',
+    actual: getTeamsList(createTestState()),
+    expected: [],
+  });
+
+  assert({
+    given: 'teams in state',
+    should: 'return an array or teams',
```

Comment:
array of teams?

Reply:
Also fixed an "array or users" from the `getUsersList` test that this was duplicated from.